client/my-sites/plugins/minimum-jet-pack-version-notice.jsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import notices from 'notices';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+import {
+	getSite,
+	isJetpackSite,
+	siteHasMinimumJetpackVersion
+} from 'state/sites/selectors';
+
+class MinimumJetpackVersionNotice extends React.Component {
+
+	static propTypes = {
+		translate: React.PropTypes.func.isRequired,
+		minimumJetpackVersionFailed: React.PropTypes.bool.isRequired,
+		site: React.PropTypes.object,
+	};
+
+	notificationHandler( { minimumJetpackVersionFailed, site, translate } ) {
+		if ( minimumJetpackVersionFailed ) {
+			notices.warning(
```

Comment:
It should be simple enough to replace the `notices.warning` calls with the corresponding dispatched action calls. It's definitely something we can tackle in another PR though :+1: